pmm/pmm2-server-release.groovy

```diff
@@ -13,7 +13,7 @@ pipeline {
     parameters {
         choice(
             // default is choices.get(0) - laboratory
-            choices: ['laboratory', 'testing'],
+            choices: ['laboratory', 'experimental'],
```

Comment:
let's just add experimental and not remove testing

Reply:
+1

Reply:
okay will do that

Reply:
Done